tests/data/hello.json

```diff
@@ -0,0 +1 @@
+Hello
```

Comment:
This is not valid JSON. Should it be?

Reply:
This is file is just used to test readFileInString so it doesn't necessarily have to be in json format. But I can see how this could be confusing. I think adding in some sort of mock json format would be good here.

Reply:
Literally just wrapping the string in double quotes makes this valid JSON. 

"Hello" is valid. Would that work?